es-ds-docs/pages/atoms/layout.vue

```diff
@@ -1,5 +1,328 @@
 <template>
     <div>
-        atoms - layout
+        <h1>
+            Layout
+        </h1>
+        <p>
+            Extended from 
+            <NuxtLink to="https://bootstrap-vue.org/docs/components/layout" target="_blank">
+                bootstrap-vue layout
+            </NuxtLink>
+        </p>
+        <h2 class="mt-200">
+            Responsive Breakpoints
+        </h2>
+        <p>
+            At each breakpoint above extra small (xs), the content is constrained to the max width listed below.
+        </p>
+        
+        <ds-responsive-table>
+            <ds-responsive-table-row
+                v-for="item in breakpointTableItems"
+                zebraStripes >
+                <ds-responsive-table-column md="1.5"
+                    v-for="field in breakpointTableFields">
+                    <template #name>
+
+                        <div 
+                            v-if="field.label == ``">
+                                &nbsp; 
+                        </div>
+                        <div v-else>
+                            {{ field.label }} 
+                        </div>
+                    </template>
+                    <template #value>
+                        <div 
+                            v-if="field.key == `label`">
+                                <strong>{{ item[field.key] }} </strong>
+                        </div>
+                        <div v-else>
+                            {{ item[field.key] }}
+                        </div>
+                    </template>
+                </ds-responsive-table-column>
+            </ds-responsive-table-row>
+        </ds-responsive-table>
+
+        <h2 class="mt-200">
+            Grid Overview
```

Comment:
Nit: this is how it was before, but can we follow our own [Typography guidelines](https://design.energysage.dev/atoms/typography) and make this sentence case like "Grid overview"?